server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/CreateAction.java

```diff
@@ -54,22 +58,24 @@
   private final Languages languages;
   private final ProfileImporter[] importers;
   private final QProfileWsSupport qProfileWsSupport;
+  private final UserSession userSession;
   private final ActiveRuleIndexer activeRuleIndexer;
 
   public CreateAction(DbClient dbClient, QProfileFactory profileFactory, QProfileExporters exporters, Languages languages,
```

Comment:
This huge number of parameters is related to Pico. I think we should not change anything about it.